### PR TITLE
User verification for reruns / deleting previous analyses

### PR DESCRIPTION
**Pull request by mariogiov**
As requested by vezzi

### CODE REVIEW

File: scripts/ngi_pipeline_start.py [hunk 1]
```diff
@@ -19,30 +20,39 @@
...
-    return _validate_dangerous_user_thing("OVERWRITE EXISTING DATA in CHARON")
-
-def validate_delete_existing():
-    return _validate_dangerous_user_thing("DELETE EXISTING DATA in CHARON")
-
-def _validate_dangerous_user_thing(action="do SOMETHING that Mario thinks you should BE WARNED about"):
-    print("DANGER WILL ROBINSON you have told this script to {action}!! Do you in fact want do to this?!?".format(action=action), file=sys.stderr)
+inflector = inflect.engine()
+
+def validate_dangerous_user_thing(action="do SOMETHING that Mario thinks you should BE WARNED about",
```
Comment (vezzi) [inline scripts/ngi_pipeline_start.py]: seriously!!!! :-D

File: scripts/ngi_pipeline_start.py [hunk 2]
```diff
@@ -110,40 +121,69 @@ def __init__(self, *args, **kwargs):
...
+    # Charon-specific arguments ('organize' or 'analyze')
+    if args.__dict__.get("force_update"):
+        args.force_update = \
+                validate_dangerous_user_thing("overwrite existing data in Charon")
+    if args.__dict__.get("delete_existing"):
+        args.delete_existing = \
+                validate_dangerous_user_thing("delete existing data in Charon")
+
+
+    import ipdb; ipdb.set_trace()
```
Comment (Galithil) [inline scripts/ngi_pipeline_start.py]: someone forgot something

File: scripts/ngi_pipeline_start.py (hunk 1 above)
Comment (guillermo-carrasco) [inline scripts/ngi_pipeline_start.py]: you really want to leave your footprint in the code don't you?

File: scripts/ngi_pipeline_start.py (hunk 2 above)
Comment (guillermo-carrasco) [inline scripts/ngi_pipeline_start.py]: yuuup..

Comment (mariogiov) [inline scripts/ngi_pipeline_start.py]: Son of a beesting

Comment (mariogiov) [inline scripts/ngi_pipeline_start.py]: Well at least it reminds everyone to use `ipdb.set_trace()` and not just boring ol' `pdb`

File: scripts/ngi_pipeline_start.py (hunk 1 above)
Comment (mariogiov) [inline scripts/ngi_pipeline_start.py]: It's the old I-wrote-it-on-an-airplane excuse again

Comment (mariogiov) [inline scripts/ngi_pipeline_start.py]: At least I have a default!

**Comment by mariogiov** [comment]
K I removed that `ipdb` statement, merge away

**Pull request merged**